tinytuya/core.py

```diff
@@ -562,14 +562,14 @@ def device_info( dev_id ):
 
 class XenonDevice(object):
     def __init__(
-            self, dev_id, address=None, local_key="", dev_type="default", connection_timeout=5, version=3.1, persist=False, dev_cid = None
+            self, dev_id, address=None, local_key="", dev_type="default", connection_timeout=5, version=3.1, persist=False, cid=None, parent=None

```

Comment:
I noticed we moved from `dev_cid` to `cid` - Makes sense as this references subdevice (child ID)  @LesTR FYI